frontend/app/src/services/axios-tranformers.ts

```diff
@@ -98,28 +72,21 @@ export const axiosNoRootCamelCaseTransformer: AxiosResponseTransformer = (
   _headers
 ) => convertKeys(data, true, true);
 
-export const setupJsonTransformer: (
-  numericKeys: string[] | null
-) => AxiosResponseTransformer = numericKeys => {
-  const reviver = createReviver(numericKeys);
-  return (data, _headers) => {
-    let result = data;
-    if (typeof data === 'string') {
-      try {
-        result = JSON.parse(data, reviver);
-        // eslint-disable-next-line no-empty
-      } catch (e: any) {}
-    }
-    return result;
-  };
+const jsonTransformer: AxiosResponseTransformer = (data, headers) => {
+  let result = data;
+  const isJson = headers?.['content-type'] === 'application/json';
```

Comment:
```const isJson = headers?.['content-type']?.includes('application/json');```

Reply:
some api have this 
<img width="479" alt="image" src="https://user-images.githubusercontent.com/26648140/206108470-f83b44cc-da57-4774-adae-e719c9f52845.png">